docker-compose.yml

```diff
@@ -36,3 +36,15 @@ services:
       interval: 30s
       timeout: 10s
       retries: 3
+  redis:
+    image: redis:5.0-alpine
```

Comment:
Why use `redis:5.0-alpine` when you could use `redis:6.0.6-alpine` any specific reason? Sorry in advance, not one to use Docker much. :sweat_smile:

Reply:
We will roll with this version for now. Please feel free to open an issue if you want to discuss this further.